exla/mix.exs

```diff
@@ -133,7 +134,23 @@ defmodule EXLA.MixProject do
     {:ok, []}
   end
 
-  defp cached_make(_) do
+  defp cached_make(args) do
+    {parsed, _args, _invalid} =
+      OptionParser.parse(args, strict: [clean_libexla_cache: :boolean, force: :boolean])
```

Comment:
Suggestion to make this a single env var called EXLA_FORCE_REBUILD, as explained above.

Reply:
Env var is probably the simpler alternative :)